examples/explicit.py

```diff
@@ -0,0 +1,31 @@
+import datetime
+import redis
+
+import lightstep
```

Comment:
So far we have been using the `lightstep` tracer as part of the examples everywhere. One thing we could do is simply use the `opentracing.tracer` instance from here, and put a comment on the examples to set that first. 

Else, at least a comment there, saying `Or replace it here the tracer of your choice`.